manage

```diff
@@ -708,22 +719,30 @@ Or on Discord: \e[94m\e[4mhttps://discord.gg/RXSTvaUvuu\e[0m" | tee -a "$logfile
   #exit the manage script with the same exit-code of the app's script
   exit $exitcode
   
-elif [ "$1" == 'update' ] && [[ "$2" =~ ^filelist:.* ]];then #list of files separated by :
-  # use the second argument to pass our list of files to update
-  #get functions from updater script
-  source "${DIRECTORY}/updater" source
-
-  updatable_apps='' updatable_files="$(echo "$2" | sed 's/^filelist://g' | tr ':' '\n')" no_status=true update_now_cli
```

Comment:
Still not a huge fan of putting this inside of the daemon when all other functionality resides as its own manage action. Why do you want it inside of the daemon? There is no signifianct time penalty for calling manage anymore.

Reply:
Because I consider `manage` to be a user-facing script for dealing with apps. 

I don't want our docs to say that manage can update files, but I also don't want it to be an undocumented feature that is accessible from command-line args.

Reply:
`daemon` mode is not supposed to be user facing at all and thats a big part of the manage script. it is accessible from command line args. so I think you are already very far away from your ideal here.

I don't see how this is different, it just needlessly complicates it.

Reply:
I have come around to including it in the daemon now that they actually use the updater script functions
https://github.com/Botspot/pi-apps/pull/2340/commits/fa729751f8a515c8cbe407736149af4b4e7970d3

I have removed the refresh from manage as a result https://github.com/Botspot/pi-apps/pull/2340/commits/29e05be3979c3203ea30d480577af23400d8cc06